edb/lang/graphql/translator.py

```diff
@@ -452,12 +456,17 @@ def _validate_fragment_type(self, frag, spread):
         self._context.path.append([Step(frag.on, frag_type)])
         self._context.include_base.append(is_specialized)
 
-    def _visit_path_filter(self, arguments):
-        f_arg = arguments.get('filter')
-        if not f_arg:
-            return None
+    def _visit_arguments(self, arguments):
+        where = offset = limit = None
+        orderby = []
+
+        for arg in arguments:
+            if arg.name == 'filter':
+                where = self.visit(arg.value)
+            elif arg.name == 'orderBy':
```

Comment:
Do we want `orderBy` or `sort`?  I've seen some implementations using the latter.  `sort` would be easier to type (shorter, no capital letters) and would read just fine too. @elprans thoughts?

Reply:
If we're considering `sort` maybe we should also consider `order`. The idea being that `order` is kinda referring to the keyword used in SQL and EdgeQL.